### PR TITLE
tig: Add new package

### PR DESCRIPTION
**Pull request by hliebel**

### CODE REVIEW

File: components/developer/tig/Makefile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Same behaviour as the 'git' package
+COMPONENT_PRE_CONFIGURE_ACTION = \
+	($(CLONEY) $(SOURCE_DIR) $(@D))
+
+# Place the config file in /etc
+CONFIGURE_OPTIONS += --sysconfdir=$(ETCDIR)
+
+CFLAGS +=	-I/usr/include/ncurses
+
+# Enable Address Space Layout Randomization
```
Comment (pyhalov) [inline components/developer/tig/Makefile]: Please, remove this no-op

File: components/developer/tig/tig.p5m [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
```
Comment (pyhalov) [inline components/developer/tig/tig.p5m:24]: Depdendency on usr/bin/git is missing

**Comment by hliebel** [comment]
Is this one ok now?

File: components/developer/tig/tig.p5m [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+depend fmri=developer/versioning/git type=require
+
+file path=etc/tigrc
```
Comment (pyhalov) [inline components/developer/tig/tig.p5m]: should it be preserve=true ?

**Pull request merged**